tests/test_designmatrix_noisepars.py

```diff
@@ -0,0 +1,46 @@
+import os
+import pint.models.model_builder as mb
+import pinttestdata
+import pytest
+
+
+@pytest.fixture()
+def model_and_toas():
+    datadir = pinttestdata.datadir
+    parfile = os.path.join(datadir, "J1713+0747_NANOGrav_11yv0_short.gls.par")
+    timfile = os.path.join(datadir, "J1713+0747_NANOGrav_11yv0_short.tim")
+    return mb.get_model_and_toas(parfile, timfile)
+
+
+def _test_designmatrix(model, toas):
```

Comment:
Better to directly express the test conditions in place rather than call this utility function - it makes it hard to tell from the test code what the expected behaviour is.

Reply:
OK.. I'll remove the utility function.